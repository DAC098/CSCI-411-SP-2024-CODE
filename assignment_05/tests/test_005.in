923 391
2
38
40
50
65
80
83
86
89
89
118
133
135
137
140
145
165
169
172
180
199
213
216
222
228
256
268
268
273
301
318
321
331
343
360
372
394
397
414
414
423
432
446
457
463
467
472
472
475
485
497
504
524
527
530
538
551
561
576
604
605
609
616
628
629
639
643
647
648
655
657
671
681
692
707
715
720
731
756
757
785
786
786
792
823
824
826
833
852
885
893
925
933
937
943
946
951
954
964
974
990
997
1001
1030
1046
1053
1054
1065
1068
1070
1091
1095
1097
1103
1114
1144
1146
1153
1168
1174
1178
1195
1203
1210
1223
1245
1245
1251
1267
1271
1273
1278
1297
1318
1346
1369
1376
1396
1413
1415
1416
1427
1439
1446
1447
1456
1457
1469
1473
1473
1480
1483
1500
1513
1517
1533
1540
1550
1557
1563
1573
1578
1578
1579
1602
1619
1623
1624
1637
1652
1685
1686
1703
1723
1725
1732
1735
1756
1756
1758
1758
1767
1778
1779
1781
1789
1831
1839
1852
1879
1884
1892
1903
1917
1920
1920
1931
1939
1940
1953
1965
1967
2003
2007
2013
2017
2026
2033
2034
2038
2043
2050
2074
2085
2091
2121
2129
2130
2135
2145
2151
2153
2155
2158
2161
2167
2173
2191
2199
2204
2207
2217
2242
2256
2264
2264
2286
2296
2315
2320
2322
2322
2335
2344
2355
2416
2431
2446
2465
2486
2500
2510
2518
2521
2572
2579
2588
2591
2614
2626
2644
2645
2646
2666
2682
2687
2689
2702
2716
2721
2721
2730
2752
2762
2775
2779
2781
2795
2810
2826
2853
2869
2877
2897
2922
2928
2933
2934
2945
2955
3002
3011
3015
3021
3059
3069
3084
3114
3115
3119
3145
3155
3159
3190
3198
3234
3248
3248
3288
3289
3302
3305
3349
3380
3388
3391
3395
3406
3431
3466
3500
3510
3516
3517
3519
3526
3532
3532
3537
3544
3550
3557
3572
3572
3579
3589
3604
3606
3628
3629
3638
3641
3654
3658
3659
3668
3685
3696
3704
3715
3732
3738
3744
3753
3753
3755
3757
3785
3792
3795
3801
3820
3825
3834
3835
3837
3840
3869
3872
3877
3880
3881
3892
3947
3956
3977
3978
3981
3995
4020
4058
4059
4059
4062
4063
4108
4138
4163
4170
4170
4182
4231
4242
4255
4274
4296
4297
4311
4313
4327
4333
4339
4342
4344
4349
4349
4359
4367
4382
4382
4390
4401
4423
4460
4471
4476
4479
4499
4499
4505
4520
4525
4527
4568
4569
4570
4580
4591
4592
4618
4620
4638
4642
4657
4670
4672
4694
4699
4701
4726
4732
4742
4746
4747
4757
4765
4774
4775
4778
4786
4804
4811
4866
4875
4876
4883
4883
4891
4914
4919
4933
4941
4949
4962
4962
4963
4974
4978
4993
4996
5004
5005
5023
5029
5047
5051
5072
5127
5197
5221
5222
5224
5240
5245
5246
5259
5271
5289
5292
5298
5308
5327
5352
5352
5367
5394
5402
5412
5423
5427
5428
5452
5459
5463
5464
5469
5478
5480
5488
5492
5507
5519
5527
5532
5544
5551
5558
5568
5575
5581
5591
5595
5603
5612
5615
5617
5630
5633
5649
5669
5680
5680
5699
5712
5725
5725
5763
5793
5797
5810
5820
5820
5822
5827
5827
5835
5836
5850
5856
5871
5887
5888
5904
5917
5917
5922
5928
5943
5967
5992
5996
5997
5997
6020
6021
6038
6063
6068
6069
6072
6080
6083
6087
6088
6114
6116
6121
6122
6139
6167
6173
6181
6193
6193
6237
6253
6258
6270
6276
6279
6296
6297
6309
6311
6317
6328
6344
6344
6345
6349
6368
6370
6402
6402
6412
6439
6457
6465
6476
6488
6498
6515
6527
6533
6544
6556
6576
6589
6594
6596
6647
6658
6658
6678
6696
6697
6717
6747
6758
6793
6800
6812
6823
6842
6863
6872
6876
6881
6889
6891
6896
6903
6908
6909
6917
6919
6920
6920
6950
6951
6962
6968
6969
6973
6978
6994
7001
7010
7011
7015
7017
7020
7037
7042
7058
7070
7071
7077
7077
7080
7095
7102
7106
7113
7117
7125
7144
7148
7165
7166
7171
7172
7193
7205
7213
7220
7251
7253
7260
7261
7266
7276
7276
7290
7293
7304
7308
7310
7331
7331
7336
7353
7358
7376
7408
7412
7413
7415
7431
7454
7469
7485
7486
7500
7521
7530
7550
7569
7584
7593
7599
7599
7609
7609
7627
7629
7639
7642
7646
7670
7674
7679
7686
7689
7694
7713
7717
7734
7751
7753
7753
7760
7765
7788
7801
7822
7827
7832
7836
7836
7839
7867
7874
7877
7891
7921
7925
7967
7975
7980
7984
8001
8033
8036
8050
8053
8057
8060
8064
8072
8073
8076
8086
8100
8125
8170
8183
8209
8268
8273
8286
8292
8341
8341
8357
8389
8394
8401
8404
8408
8414
8415
8423
8426
8442
8446
8518
8523
8536
8540
8545
8549
8553
8557
8563
8564
8580
8580
8592
8598
8618
8629
8636
8644
8649
8654
8665
8679
8690
8698
8704
8704
8728
8730
8742
8746
8752
8762
8762
8766
8774
8781
8812
8821
8832
8836
8852
8856
8876
8890
8906
8911
8927
8936
8981
9003
9017
9023
9043
9066
9066
9098
9106
9116
9118
9124
9130
9136
9156
9185
9199
9214
9214
9216
9227
9244
9246
9281
9282
9294
9308
9316
9324
9333
9333
9340
9360
9384
9393
9399
9400
9402
9410
9445
9451
9455
9499
9507
9519
9546
9546
9552
9579
9646
9661
9675
9688
9700
9712
9730
9740
9747
9750
9752
9768
9792
9829
9833
9845
9846
9892
9896
9901
9909
9917
9922
9939
9950
9967
9967
9981
9987
9992
