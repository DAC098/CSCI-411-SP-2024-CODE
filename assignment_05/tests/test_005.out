Enter the number of hotels and the ideal number of miles to travel per day: Enter 923 hotel distances each on a separate line: Hotels to visit: 37 81 119 158 197 239 268 297 320 367 392 430 472 496 537 579 613 643 694 731 767 795 834 861 893 923 
Total penalty: 2772
